Web/starter_project_iconics/pages/blog/index.tsx

```diff
@@ -24,12 +29,14 @@ const blogs = () => {
       </div>
 
```

Comment:
some of the spacing is inconsistent, so use prettier to format it, you can use the extension if you want.